packages/iris-grid/src/IrisGridTableModelTemplate.ts

```diff
@@ -636,6 +647,49 @@ class IrisGridTableModelTemplate<
     return 'left';
   }
 
+  dataBarOptionsForCell(column: ModelIndex, row: ModelIndex): DataBarOptions {
+    const format = this.formatForCell(column, row);
+    assertNotNull(format);
+    const {
+      axis,
+      direction,
+      max,
+      min,
+      negativeColor,
+      opacity,
+      positiveColor,
+      valuePlacement,
+      value,
+      // markers,
+    } = format.formatDataBar;
+    const posColor = positiveColor ?? theme.positiveBarColor;
```

Comment:
Was thinking of passing in theme the same it is passed into `colorForCell`. Brings up complications in DataBarGridModel though since that IrisThemeType is not there.

Reply:
We shouldn't be using the `theme` imported from `IrisGridTheme`, since you can pass in a custom theme as a prop and we need to respect that.
We don't need to define `IrisThemeType` in `DataBarGridModel` either, just use `GridTheme` type (like we do for `colorForCell`) and pass that in. `IrisGridTheme` extends `GridTheme`.